gen/dcos-config-windows.yaml

```diff
@@ -0,0 +1,759 @@
+package:
+  - path: c:/etc/dcos-metrics-config.yaml
+    content: |
+  - path: /etc/dcos-metrics.env
+    content: |
+      DCOS_METRICS_CONFIG_PATH=/opt/mesosphere/etc/dcos-metrics-config.yaml
+  - path: c:/pkginfo.json
+    content: |
+      {
+        "environment": {
+          "MESOS_IP_DISCOVERY_COMMAND": "/opt/mesosphere/bin/detect_ip.ps1",
+          "PROVIDER": "{{ provider }}"
+        }
+      }
+  - path: c:/bin/detect_ip.ps1
+    permissions: "0755"
+    content: {{ ip_detect_contents }}
+  - path: c:/bin/detect_ip_public.ps1
+    permissions: "0755"
+    content: {{ ip_detect_public_contents }}
+  - path: c:/etc/check_time.env
+    content: |
+        ENABLE_CHECK_TIME={{ check_time }}
+{% switch licensing_enabled %}
+{% case "true" %}
+  - path: c:/etc/license.txt
+    permissions: "0600"
+    content: {{ license_key_contents }}
+{% case "false" %}
+{% endswitch %}
+  - path: c:/check/check_ip.ps1
+    permissions: "0755"
+    content: |
+      #!/bin/bash
```

Comment:
I am actually very serious about having time to do this properly. The more I split off the more breaks. As you saw above. I stripped a lot of stuff out and that was not liked. I am doing my best so we can get to do this properly. All this stripping out stuff is making it harder to actually do it properly.

Reply:
I will strip out more in a hope I don't break more of pkgpanda code.

Reply:
Should be possible to just have a single placeholder config file. If any tests fail because they're looking for specific config, sounds like that's a platform-specific test that should be skipped on Windows.